playground.sql

```diff
@@ -1,5 +1,15 @@
 \c nc_news
 
-SELECT * FROM articles
-WHERE article_id = 1;
-SELECT * FROM information_schema.tables;
+SELECT articles.article_id, title, topic, articles.author, articles.created_at, article_img_url, COUNT(comments.article_id) AS comment_count, SUM(comments.votes) AS votes
+FROM articles
+LEFT JOIN comments 
+ON articles.article_id = comments.article_id
+GROUP BY articles.article_id, comments.article_id
+ORDER BY articles.article_id DESC;
+
+-- SELECT comments.article_id, COUNT(*) AS comment_count, SUM(comments.votes) AS votes, articles.title, articles.topic, articles.author, articles.created_at, articles.article_img_url
```

Comment:
We shouldn't really push this sql file as it's only there for your testing purposes